.vsts-pipelines/builds/PRBuild.yml

```diff
@@ -0,0 +1,232 @@
+phases:
+- template: ../templates/phases/default.yml
+  parameters:
+    name: 'UniverseCoherence'
+    buildArgs: >
+      $(PipebuildArgs)
+      /p:SkipTests=true
+      /p:Configuration=Release
+      /p:BuildNumber=$(Build.BuildNumber)
+
+#BEGIN SharedFx
+- template: ../templates/phases/default.yml
+  parameters:
+    name: SharedFxWin64
+    dependsOn: UniverseCoherence
+    beforeBuild:
+    - task: DownloadBuildArtifacts@0
```

Comment:
Much of this appears to be redundant.  Download, copy, build.  Maybe consider a design where you are passing unique configurations to a template and the template uses those configuration values to perform these tasks.  The general model I tried to follow in Arcade was `.vsts-ci.yml (configuration) -> eng\build.yml (build steps) -> eng\common\templates\… (underlying common goo)`